### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ashkrm**
added thumbnail on hover details, some UI changes and fixed issues

### CODE REVIEW

File: src/components/MovieCard.vue [hunk 1]
```diff
@@ -1,23 +1,76 @@
...
-export default ({
+  export default ({
     props: ['id', 'imgSrc'],
     methods: {
-        routerFunction() {
-            let route = this.$router.resolve({ path: "/details/"+this.id});
-            window.open(route.href);
-        }
+      routerFunction() {
+        let route = this.$router.resolve({
```
Comment (lipi17dpatnaik) [inline src/components/MovieCard.vue:20]: why not const?

File: src/components/MovieCard.vue [hunk 2]
```diff
@@ -1,23 +1,76 @@
...
+    transform: scale(1.2);
+  }
+  img:hover + .my-content {
+    z-index: 10;
+  }
+  .my-content {
+    background-image: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.6));
+    color: white;
+    width: 100%;
+    position: absolute;
```
Comment (lipi17dpatnaik) [inline src/components/MovieCard.vue:54]: why not relative?

File: src/App.vue [hunk 3]
```diff
@@ -29,8 +29,9 @@
 </script>
 
 <style>
-  html,
-  body {
+  .wrapper {
+    margin: 0;
+    padding: 0;
     width: 100vw;
     min-height: 100vh;
```
Comment (lipi17dpatnaik) [inline src/App.vue:36]: why this?

**Review by lipi17dpatnaik — APPROVED** [review]

File: src/components/MovieCard.vue (hunk 2 above)
Comment (Ashkrm) [reply src/components/MovieCard.vue:54]: i'm setting the position relative to its parent so the parent has position: relative and this has position:absolute

**Pull request merged**